examples/windows/windows-perfmon-proc-owner.yml

```diff
@@ -0,0 +1,13 @@
+integrations:
+  - name: nri-flex # We're telling the Infra agent to run Flex!
+    interval: 60s
+    timeout: 45s
```

Comment:
curious: is it necessary to extend this past the default 30s if the Flex timeout below is also 30s?

Reply:
probably not